roles/gateway_update_otk_conf/README.md

```diff
@@ -0,0 +1,52 @@
+Update Hostnames used by OTK
+============================
+
+This role updates the hostnames used by OTK using the values configured under this role's `vars/main.yml`:
+- Gateway cluster.hostname cluster property
+- OTK's JDBC connection URLs
+- Callback URLs
+
+Requirements
+------------
+
+Before running this role:
+- OTK should already be migrated to the destination database, which is the internal MySQL database running on a Gateway.
+- Ensure the Ansible controller can modify the OTK database
+  - Note: otk variables correspond to the value of the otkdb_name, otkdb_user and otkdb_userpwd variables, respectively (see Group Variables section)
+  - e.g. CREATE USER <otk user>@'localhost' IDENTIFIED BY '<otk user pass>';
+         GRANT SELECT, INSERT, DELETE, UPDATE ON <otk db name>.* TO <otk user>@'localhost';
+
```

Comment:
in the roles gateway_import_database there are steps grant otk user privileges maybe you don't need to do this

Reply:
hmm I ran that role before trying this one, and I got mysql errors until I manually set the grant for localhost

Reply:
yeah that import db one grants on the inventory_hostname node which is not the controller